tests/main.tf

```diff
@@ -1,26 +1,27 @@
 module "primary-cluster" {
-  name               = "primary-cluster"
-  source             = "../."
-  region             = "${var.region}"
-  zones              = "${var.zones}"
-  project            = "${var.project}"
-  environment        = "${terraform.workspace}"
-  min_master_version = "${var.kube_version}"
```

Comment:
Pls leave one test with `min_master_version` and `version` in node_pools - you can use different node pool for that with spefic version.

Reply:
👍